lib/cartopy/crs.py

```diff
@@ -1955,6 +1955,83 @@ def y_limits(self):
                                           semiminor_axis=6371007.181))
 
 
+class EquidistantConic(Projection):
+    """
+    An Equidistant Conic projection.
+
+    """
+
+    def __init__(self, central_longitude=0.0, central_latitude=0.0,
+                 false_easting=0.0, false_northing=0.0,
+                 standard_parallels=(20.0, 50.0), globe=None):
+        """
+        Kwargs:
```

Comment:
Not really your fault, but the docstring format has been updated since this branch was created.

See https://github.com/SciTools/cartopy/pull/998/files#diff-6dfcada343269e6678a777c4f9a5223d for reference.